### DISCUSSION TITLE
How to replace date picker?

### DISCUSSION THREAD

**Discussion by Nimaj1994** [ckan/ckan#7811] [Q&A]
Hi everyone,
I’m currently working with ckan 2.10 and also I install ckanext-scheming extension. When we add a metadata field with date preset, it shows a date picker which is not a part of scheming extension and it is set by the ckan itself. I need to change this date picker to another one. How can I do that and where should I look?

**Comment by wardi** [comment]
@Nimaj1994 you can create your own [form_snippets](https://github.com/ckan/ckanext-scheming#form_snippet) templates and use them in your schema instead of the default ones, or completely override the `scheming/form_snippets/datetime.html` in your theme extension (just make sure your theme plugin comes before the scheming one in the plugins list)
